tests/bootstrap.php

```diff
@@ -83,6 +83,5 @@
     unset($db_config);
 }
 
-\LibreNMS\Util\OS::updateCache(true); // Force update of OS Cache
-
 app()->terminate(); // destroy the bootstrap Laravel application
+app()->make('librenms-config');
```

Comment:
Not sure this is needed, however, I'm unsure if tests clear singleton instances or not.

Reply:
This was a test to try and figure out why we were getting the test errors.